Documentation/Storage-Configuration/Object-Storage-RGW/object-storage.md

```diff
@@ -184,6 +185,80 @@ To consume the object store, continue below in the section to [Create a bucket](
 Modify the default example object store name from `my-store` to the alternate name of the object store
 such as `store-a` in this example.
 
+### Create Local Object Store(s) with pool placements
+
```

Comment:
@travisn I thought we were going to start out with this marked experimental. Does that sound right?

Reply:
done